lisa/sut_orchestrator/baremetal/cluster/cluster.py

```diff
@@ -93,5 +93,14 @@ def _fill_capability(self, node_capability: schema.NodeSpace) -> None:
             items=[
                 schema.FeatureSettings.create(features.SerialConsole.name()),
                 schema.FeatureSettings.create(features.StartStop.name()),
+                schema.FeatureSettings.create(features.SecurityProfile.name()),
```

Comment:
Change this line should work without below reload.
`features.SecurityProfileSettings(),`